docs/permission/overview.md

```diff
@@ -0,0 +1,41 @@
+---
+id: overview
+title: Permission Framework Overview
+description: A high level overview of the Backstage permission framework
+---
+
+[The previous section](../auth/index.md) covered the various _authentication_ methods of Backstage, but Backstage can also _authorize_ specific data, APIs, or interface actions - meaning that Backstage has the ability to enforce rules about what actions are allowed for a given user of a system.
```

Comment:
nit: it's not just permission to perform actions, but also ability to read resources.

Reply:
Revised to reflect this.